src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/repository/Node.kt

```diff
@@ -296,6 +298,18 @@ class Node(val repository: Repository, val path: String) : Serializable {
     val json: String
         get() = Formats.toJson(this)
 
+    /**
+     * Download node as CRX package.
+     */
+    fun download(options: PackageDefinition.() -> Unit = {}): File {
+        val node = this
+        return repository.sync.packageManager.download {
+            archiveBaseName = Formats.manglePath(node.name)
```

Comment:
this is for correctly saving converting `rep:policy` to `_rep_policy`/  mangling